galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.docker.manager/src/main/java/dev/galasa/docker/internal/DockerRegistryImpl.java

```diff
@@ -265,8 +267,18 @@ private boolean retrieveRealm(DockerImageImpl image) throws DockerManagerExcepti
 				}
 			}
 			throw new DockerManagerException("Failed to authenticate, and authentication is required.");
-		} catch (HttpClientException | MalformedURLException | JsonSyntaxException e) {
-			throw new DockerManagerException("Failed to connect to registry", e);
+		} catch (HttpClientException e) {
+			logger.trace("HttpClientException, with message: " + e.getMessage());
+			StringWriter sw = new StringWriter();
+			PrintWriter pw = new PrintWriter(sw);
+			logger.trace("HttpClientException stack trace:\n" + sw.toString());
```

Comment:
dont need these traces,     include the information in the throw,  same for all the the others in this try block